ui/src/app/applications/components/application-deployment-history/revision-metadata-rows.tsx

```diff
@@ -5,6 +5,50 @@ import {ApplicationSource, RevisionMetadata, ChartDetails} from '../../../shared
 import {services} from '../../../shared/services';
 
 export const RevisionMetadataRows = (props: {applicationName: string; applicationNamespace: string; source: ApplicationSource; index: number; versionId: number | null}) => {
+    if (props?.source?.repoURL?.startsWith('oci://')) {
+        return (
+            <DataLoader
+                input={props}
+                load={input =>
+                    // TODO: Needs real impl
```

Comment:
Can you note how real impl would look like roughly?

Reply:
The actual impl is actually in place :D

Just forgot to remove that part, and just added a few more ui tweaks